docs/docs/twig/mount.md

```diff
@@ -0,0 +1,20 @@
+# The mount statement
```

Comment:
Don't we have any existing documentation for the `mount` statement?
cc @jamesb3ll @tomdwyer

Reply:
Don't think so, we had something about `include`, `mount` and `embed` somewhere in the old docs, but it wasn't explained that well and I'm not sure where that is now.